nests.json

```diff
@@ -224,6 +252,18 @@
       "status": "experimental",
       "category": "core",
       "readme": "https://raw.githubusercontent.com/anboto/Xlnt/main/README.md"
+    },
+	{
+      "name": "Npy",
+      "packages": [
+        "Npy",
+		"plugin/libnpy"
+      ],
+      "description": "Npy / Npz library wrapper",
```

Comment:
Would be good to expand Npy and Npz shortcut.

Reply:
Sorry, I don't understand you.